doc/installation.md

```diff
@@ -28,7 +28,7 @@ $ chmod a+x kerl
 $ mv kerl /usr/bin
 $ kerl list releases
 $ kerl build 18.2 r18.2 # this takes a while
-$ kerl install r18.2 /usr/lib/erlang
+$ kerl install r18.2 /usr/local/erlang
```

Comment:
i'm with @jamesaimonetti on this, its a dev thing. the released package already runs without the need for erlang, it doesn't use it even if installed.